src/helpers/ListingHelper.js

```diff
@@ -4,11 +4,15 @@ import { useIntl, defineMessages } from 'react-intl';
 
 import { flattenToAppURL } from '@plone/volto/helpers';
 import { When } from '@plone/volto/components/theme/View/EventDatesInfo';
-import { viewDate } from 'design-comuni-plone-theme/helpers';
+import {
+  viewDate,
+  getRealStartAndEndWithRecurrence,
+} from 'design-comuni-plone-theme/helpers';
 import {
   getCalendarDate_extend,
   getEventRecurrenceMore_extend,
 } from 'design-comuni-plone-theme/helpers/ListingHelper_extend';
+import { rrulestr } from 'rrule';
```

Comment:
Di questo bisogna fare lazy loading. In questo caso significa che `rrulestr` dovrà essere passato come parametro alla funzione `getCalendarDate`, ma non abbiamo scelta. Già viene passato come paramentro a `getRealStartAndEndWithRecurrence` per lo stesso motivo, quindi bisogna farlo anche qui dato che serve.

Reply:
Ma posso non essendo un componente?

Reply:
Non direttamente lì, devi fare lazy loading di rrule nel componente che usa la funzione, estrarre rrulestr e passarlo alla funzione. Se cerchi rrulestr nel codice trovi parecchi altri esempi

Reply:
Ok, fatto, ma ora si sbragano tutti i test dove c'e' del lazy loading. Usare 
```
jest.mock('@plone/volto/helpers/Loadable/Loadable');
beforeAll(
  async () =>
    await require('@plone/volto/helpers/Loadable/Loadable').__setLoadables(),
);
```
come in altre parti dove ci si aspetta delle lib lazy non porta nessun effetto